concepts/05 Widgets/Tabs/15 Control the Behavior.md

```diff
@@ -1,64 +1,444 @@
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Tabs } from 'devextreme-react/tabs';
+
+    const tabs = [
+        { text: "User" },
+        { text: "Comment" },
+        // ...
     ];
 
+    class App extends React.Component {
+        render() {
+            return (
+                <Tabs
+                    items={tabs}
+                    selectionMode="multiple"
+                />
+            );
+        }
+    }
+
+    export default App;
+
+---
+
+If you need a tab to be preselected or to select it programmatically, pass its index in the data source array to the [selectedIndex](/api-reference/10%20UI%20Widgets/CollectionWidget/1%20Configuration/selectedIndex.md '/Documentation/ApiReference/UI_Widgets/dxTabs/Configuration/#selectedIndex') option.
+
+---
+##### jQuery
+
+    <!--JavaScript-->
     $(function() {
         $("#tabsContainer").dxTabs({
-            items: tabs,
-            // Preselects the tab with index 1
+            items: [
+                { text: "User" },
+                { text: "Comment" },
+                // ...
+            ],
             selectedIndex: 1
         });
     });
 
-<!---->
+##### Angular
+
+    <!--HTML-->
+    <dx-tabs
+        [items]="tabs"
+        [selectedIndex]="1">
+    </dx-tabs>
+
+    <!--TypeScript-->
+    import { DxTabsModule } from "devextreme-angular";
+    // ...
+    export class AppComponent {
+        tabs = [
+            { text: "User" },
+            { text: "Comment" },
+            // ...
+        ];
+    }
+    @NgModule({
+        imports: [
+            // ...
+            DxTabsModule
+        ],
+        // ...
+    })
+
+##### Vue
+
+    <!--tab: App.vue-->
+    <template>
+        <DxTabs
+            :items="tabs"
+            :selected-index="1" /> 
+    </template>
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxTabs from "devextreme-vue/tabs";
+
+    export default {
+        components: {
+            DxTabs
+        },
+        data() {
+            return {
+                tabs: [
+                    { text: "User" },
+                    { text: "Comment" },
+                    // ...
+                ]
+            };
+        }
+    };
+    </script>
+
+##### React
 
-    <!--JavaScript-->// Selects the tab with index 0
-    $("#tabsContainer").dxTabs("option", "selectedIndex", 0);
+    <!--tab: App.js-->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Tabs } from 'devextreme-react/tabs';
+
+    const tabs = [
+        { text: "User" },
+        { text: "Comment" },
+        // ...
+    ];
+
+    class App extends React.Component {
+
+        render() {
+            return (
+                <Tabs
+                    items={tabs}
+                    selectedIndex={1}
```

Comment:
In this example, we should show bindings similar to those that we show for `selectedItems`.